### ISSUE TITLE
Question on Reported FLOPs

### ISSUE THREAD

**Issue by ooodragon94** [liuzechun/ReActNet#17]
Hi
I was wondering on how you calculated your model's FLOPs.
May I ask for flop calculation code?
I have tried out myself, and it seems like it gives the reported result only when I don't calculated full precision modules (e.g. Batchnorm2d, PReLU, AvgPool)

thanks for your help!

**Comment by jawaechan** [comment]
Hi, 
   Could you share the code of calculating the BOPs ?

Thanks

**Comment by ooodragon94** [comment]
> Hi,
> Could you share the code of calculating the BOPs ?
> 
> Thanks

Used following repo
https://github.com/sovrasov/flops-counter.pytorch/blob/master/ptflops/flops_counter.py

**Comment by jawaechan** [comment]
> > Hi,
> > Could you share the code of calculating the BOPs ?
> > Thanks
> 
> Used following repo
> https://github.com/sovrasov/flops-counter.pytorch/blob/master/ptflops/flops_counter.py

It seems this tool can not count binary operators.

**Comment by ooodragon94** [comment]
you will have to implement that

**Comment by liuzechun** [comment]
Hi thanks for your interest in our work. We calculate the BOPs by formulas. We don't have the code to read the BOPs from the network. The FLOPs in the full precision modules (e.g. Batchnorm2d, PReLU, AvgPool) are not calculated since they are very small compared to the computation in FC and Conv layers.
